### ISSUE TITLE
Sheet music index tree

### ISSUE THREAD

**Issue by Splines** [vincerubinetti/vincentrubinetti.com#2]
Hey Vincent Rubinetti 👋 Your website is visually stunning and very clean, I like your design. I was wondering if you designed the [sheets website](https://vincentrubinetti.com/sheets/) index tree on your own as well. I haven't found it in this repo though (only a link to that site). Do you have it available as open-source repo? All the best, Dominic.

**Comment by vincerubinetti** [comment]
Hi there, thank you for the kind words. I tend to just dump "extra" things like sheet music/stems/etc directly onto my server without committing them to this repo, because they change a lot, aren't as important, and possibly shouldn't in a repo (might seem like I'm using GitHub for regular file storage which you're not supposed to do).

That listing you linked is really just the [Apache default file listing](https://stackoverflow.com/questions/13736936/html-directory-listing-formatting) (you've probably seen this before) with some basic custom styles/functionality appended on, which you can see in [`listing-header.html`](https://github.com/vincerubinetti/vincentrubinetti.com/blob/main/public/listing-header.html). 

Basically, my hosting service Fast Comet uses Apache, and Apache I guess generates an `index.html` for any folder on the server that doesn't have one. Then, in my `.htaccess` file, I tell Apache to add the contents of `listing-header.html` to each `index.html`.

I'm sure there are better, more modern ways to do this. I considered making some more elaborate, fully customizable solution. But ultimately I opted for the simple quick fix, given I didn't think many people would be looking at those pages anyway.

**Comment by Splines** [comment]
Wow, such a clever use of [`HeaderName`](https://httpd.apache.org/docs/2.4/mod/mod_autoindex.html#headername), didn't even know that this option existed. This looks so much better than the default Apache file listing. Great "simple quick fix" that doesn't even need a more complicated solution; thanks for sharing.
